packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -279,7 +277,7 @@ export async function oxygenDeploy(
       const choices = [
         ...deploymentData.environments.map(({name, branch}) => ({
           label: name,
-          value: branch,
+          value: name === 'Preview' ? 'shopify-preview-environment' : branch,
```

Comment:
Just tried on my production store and it is possible to have a custom environment named 'Preview', just want to make sure this implementation would differentiate the 2.
![image](https://github.com/Shopify/hydrogen/assets/9320265/0d1f135b-192b-49bb-a39f-51b6dcf76160)

Reply:
Thanks! I thought this was reserved, will switch to using env type (which I should have done from the get-go!).